### ISSUE TITLE
Error on <TAB>

### ISSUE THREAD

**Issue by arozehnal** [jceb/vim-orgmode#363]
I'm using vim 8.2 on Win 10.  I already knew about the issue [https://github.com/jceb/vim-orgmode/issues/176](url) before installing org-mode.

I prepared minimalist .vimrc:
set nocompatible
filetype plugin indent on																					
call plug#begin('~/vimfiles/plugged')
Plug 'tpope/vim-speeddating'
Plug 'mattn/calendar-vim'
let g:calendar_mruler = 'Led,Úno,Bre,Dub,Kve,Cer,Crc,Srp,Zár,Ríj,Lis,Pro'
let g:calendar_wruler = 'Ne Po Út St Ct Pá So'

Plug 'jceb/vim-orgmode'																							
call plug#end()

but I still have a problem with using <TAB> as issue 176.

`:ver

+acl                +eval               +multi_lang         -tag_any_white
+arabic             +ex_extra           +mzscheme/dyn       +tcl/dyn
+autocmd            +extra_search       +netbeans_intg      -termguicolors
+autochdir          -farsi              +num64              +terminal
+autoservername     +file_in_path       +ole                -termresponse
+balloon_eval       +find_in_path       +packages           +textobjects
-balloon_eval_term  +float              +path_extra         +textprop
+browse             +folding            +perl/dyn           -tgetent
++builtin_terms     -footer             +persistent_undo    +timers
+byte_offset        +gettext/dyn        +popupwin           +title
+channel            -hangul_input       -postscript         +toolbar
+cindent            +iconv/dyn          +printer            +user_commands
+clientserver       +insert_expand      +profile            +vartabs
+clipboard          +job                +python/dyn         +vertsplit
+cmdline_compl      +jumplist           +python3/dyn        +virtualedit
+cmdline_hist       +keymap             +quickfix           +visual
+cmdline_info       +lambda             +reltime            +visualextra
+comments           +langmap            +rightleft          +viminfo
+conceal            +libcall            +ruby/dyn           +vreplace
+cryptv             +linebreak          +scrollbind         -vtp
+cscope             +lispindent         +signs              +wildignore
+cursorbind         +listcmds           +smartindent        +wildmenu
+cursorshape        +localmap           +sound              +windows
+dialog_con_gui     +lua/dyn            +spell              +writebackup
+diff               +menu               +startuptime        -xfontset
+digraphs           +mksession          +statusline         -xim
+directx            +modify_fname       -sun_workshop       +xpm_w32
-dnd                +mouse              +syntax             -xterm_save
-ebcdic             +mouseshape         +tag_binary
+emacs_tags         +multi_byte_ime/dyn -tag_old_static
   systémový vimrc soubor: "$VIM\vimrc"
     uživatelský vimrc soubor: "$HOME\_vimrc"
 druhý uživatelský vimrc soubor: "$HOME\vimfiles\vimrc"
 třetí uživatelský vimrc soubor: "$VIM\_vimrc"
      uživatelský exrc soubor: "$HOME\_exrc"
  druhý uživatelský exrc soubor: "$VIM\_exrc"
  systémový gvimrc soubor: "$VIM\gvimrc"
    uživatelský gvimrc soubor: "$HOME\_gvimrc"
druhý uživatelský gvimrc soubor: "$HOME\vimfiles\gvimrc"
třetí uživatelský gvimrc soubor: "$VIM\_gvimrc"
       defaults file: "$VIMRUNTIME\defaults.vim"
    systémový soubor s menu: "$VIMRUNTIME\menu.vim"
Překlad: cl -c /W3 /nologo  -I. -Iproto -DHAVE_PATHDEF -DWIN32  -DFEAT_CSCOPE -DFEAT_
TERMINAL -DFEAT_SOUND -DFEAT_NETBEANS_INTG -DFEAT_JOB_CHANNEL   -DFEAT_XPM_W32   -DWI
NVER=0x0501 -D_WIN32_WINNT=0x0501 /MP -DHAVE_STDINT_H /Ox /GL -DNDEBUG /arch:IA32 /Zl
 /MT -DFEAT_OLE -DFEAT_MBYTE_IME -DDYNAMIC_IME -DFEAT_GUI_MSWIN -DFEAT_DIRECTX -DDYNA
MIC_DIRECTX -DFEAT_DIRECTX_COLOR_EMOJI -DDYNAMIC_ICONV -DDYNAMIC_GETTEXT -DFEAT_TCL -
DDYNAMIC_TCL -DDYNAMIC_TCL_DLL=\"tcl86t.dll\" -DDYNAMIC_TCL_VER=\"8.6\" -DFEAT_LUA -D
DYNAMIC_LUA -DDYNAMIC_LUA_DLL=\"lua53.dll\" -DFEAT_PYTHON -DDYNAMIC_PYTHON -DDYNAMIC_
`

 :python import sys; print(sys.version)
2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:19:30) [MSC v.1500 32 bit (Intel)]

Anny suggestion?

**Comment by arozehnal** [comment]
I made this attempt ...
I have WSL running in Win10, so, I installed into it vim and this plugins:
calendar-vim
vim-solarized8
vim-speeddating

and I don't have a problem with <TAB>. org-mode works.

Then I noticed that in the listing 'map' from gvim Win build all mapping for org-mode is missing. 
There is only mapping for:
calendar-vim
vim-speeddating

I can't explain this because PlugStatus shows that org-mode is installed.  :-(
